packages/api/src/learning-materials/learning-materials.controller.ts

```diff
@@ -1,11 +1,11 @@
 import { Controller, Get, Post, UseGuards } from '@nestjs/common';
+import { JwtAuthGuard } from 'src/auth/jwt/jwt-auth.guard';
```

Comment:
💭 Use a relative path as our tests and some scripts doesn't resolve paths that start with `src`